packages/transit-vehicle-overlay/src/components/markers/ModeRectangles/index.js

```diff
@@ -46,3 +46,7 @@ export const DetailedRectangle = utils.makeRotatedMarker(
   },
   getRectangleSize
 );
+
+// Rename styled components for export
+const Styled = StyledRectangle;
```

Comment:
I would `import * as Styled` and remove this statement and comment above.

Reply:
See previous comment https://github.com/opentripplanner/otp-ui/pull/277#discussion_r709920853